src/apps/vpn/tasks/sendfeedback/tasksendfeedback.cpp

```diff
@@ -6,12 +6,12 @@
 
 #include <QJsonObject>
 
+#include "app.h"
 #include "appconstants.h"
+#include "env.h"
```

Comment:
So I can learn, why do we need to add this line here?

Reply:
It appears to be used further down on line 48:

```
{"versionString", Env::versionString()},
```

Reply:
Thank you!